images/kube-vip/README.md

```diff
@@ -0,0 +1,71 @@
+<!--monopod:start-->
+# kube-vip
+| | |
+| - | - |
+| **OCI Reference** | `cgr.dev/chainguard/kube-vip` |
+
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/kube-vip/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
+* [Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+<!--overview:start-->
+Kubernetes Control Plane Virtual IP and Load-Balancer
+<!--overview:end-->
+
+<!--getting:start-->
+## Download this Image
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/kube-vip:latest
+```
+<!--getting:end-->
+
+<!--body:start-->
+## Usage
+
+## Deployment
+
+kube-vip has [documentation](https://kube-vip.io/docs/installation) covering
+deployments. They don't make any reference to helm charts, although there seems
+to be one published separately (see section below).
+
+The deploy instructions differ depending on your k8s environment. Refer to
+the upstream docs](https://kube-vip.io/docs/installation) for the correct set
+of instructions. Be sure to replace the image reference in any manifests to the
+Chainguard image.
+
+For reference, [here are some steps](https://github.com/chainguard-images/images/blob/main/images/kube-vip/TESTING.md)
+we followed to setup a local test environment using a local k3d cluster.
+
+### Helm
+
+kube-vip publishes a [helm chart](https://artifacthub.io/packages/helm/kube-vip/kube-vip)
+for deploying the image. **However**, they have commented out 'tag' in the
+[values.yaml](https://github.com/kube-vip/helm-charts/blob/5f24093899db53f7c103bd95b0e41a112bbfb72b/charts/kube-vip/values.yaml#L8):
+
+```bash
+image:
+  repository: ghcr.io/kube-vip/kube-vip
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: "v0.7.0"
+```
+
+If you wish to use this helm chart, you'll need to provide a custom values.yaml
+file, which is updated to use the chainguard image and latest tag:
+
+```bash
+helm repo add kube-vip https://kube-vip.github.io/helm-charts/
+helm install my-kube-vip kube-vip/kube-vip -f my-custom-values.yaml
```

Comment:
Can we clarify what the content of `my-custom-values.yaml` is going to be? Couldn't we pass something like:
```
  --set image.repository="cgr.dev/chainguard/kube-vip
```

Reply:
The helm chart doesn't expose the 'tag' property, so a custom values.yaml is needed in any event, so both image and tag would be updated in that file.

I did start creating an example file but it's got quite a lot of settings in it - I figured re-directing them to the upstream docs would be better, than them copying ours like-for-like and adding wrong settings elsewhere